AW_MiniProject1_TextMining.ipynb

```diff
@@ -0,0 +1,111 @@
+{
```

Comment:
Overall, good job!

A few comments:
- In the future, please upload a .py file instead of a Jupyter notebook, as it's much easier for us to read and critique.
- Small thing: Try to be consistent with your spacing. In between functions, you should generally have 2 lines of space.
- When you do have a Jupyter Notebook, be sure to run all cells before pushing to github, because then we can see all of your output. (I can't see the plot you made, but it looks like you attached it elsewhere so you're all good)